translations/translationtool/src/translationtool.php

```diff
@@ -302,12 +313,45 @@ private function createFakeFileForAppInfo() {
 			$content .= '$l->t(' . $this->escape($string) . ');' . PHP_EOL;
 		}
 		
-		file_put_contents($this->dummyFileName, $content);		
+		file_put_contents($this->fakeAppInfoFile, $content);
+	}
+
+	private function createFakeFileForVueFiles() {
+		$fakeFileContent = '';
+
+		foreach ($this->findTranslatableFiles(['.vue']) as $vueFile) {
+			$vueSource = file_get_contents($vueFile);
+			if ($vueFile === false) {
```

Comment:
I guess you mean `$vueSource` or do I miss something here?

Reply:
Correct!